README.md

```diff
@@ -791,8 +777,7 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   hash = {}
   ~~~
 
-* Prefer the literal array syntax to `%w`, except when it reads substantially
-  more clearly in context.
```

Comment:
"except when..." is implicit not just for the rule but for pretty much everything in this style guide.